onnxruntime/python/tools/quantization/__init__.py

```diff
@@ -0,0 +1,2 @@
+from onnxruntime.quantization.quantize import quantize
```

Comment:
from . import

Reply:
keep consistent with other modules to use absolute path